_includes/code/python/howto.configure.rbac.roles.py

```diff
@@ -0,0 +1,125 @@
+import weaviate
+
+# START AdminClient 
+from weaviate.classes.init import Auth
+
+# Connect to Weaviate as Admin
+admin_client = weaviate.connect_to_local(auth_credentials=Auth.api_key("admin-key"))
+# END AdminClient
+
+# START CreateRole
+from weaviate.classes.rbac import Permissions
+
+admin_client.roles.create(
+    role_name="devrel", permissions=<LIST-OF-PERMISSIONS>
+)
+# END CreateRole
+
+# START AddRoleAtCreate
+from weaviate.classes.rbac import Permissions
+
+permissions = [
+    Permissions.collections(
+      collection="Test_*", 
+      read_config=True, 
+      create_collection=False, 
+      delete_collection=False,
+  ),
+    Permissions.collections(
+      collection="Test_DevRel", 
+      read_config=True, 
+      create_collection=True, 
+      delete_collection=True,
+  ),
+]
+
+admin_client.roles.create(
+    role_name="devrel", permissions=permissions
+)
+# END AddRoleAtCreate
+
+# START AddRoles
+from weaviate.classes.rbac import Permissions
+
+permissions = [
+    Permissions.data(collection="Test_DevRel", read=True, create=True),
+    Permissions.data(collection="Test_*", read=True, create=False)
+]
+
+admin_client.roles.add_permissions(permissions=permissions, role_name="devrel")
+# END AddRoles
+
+# START ManageRoles
+from weaviate.classes.rbac import Permissions
+
+permissions = [
+    Permissions.roles(role_names="devrel", read=True, manage=True),
+    Permissions.roles(role_names="devrel-*", read=True, manage=False)
+]
+
+admin_client.roles.create(role_name="devrel-admin", permissions=permissions)
+# END ManageRoles
+
+# START ClusterAndNodePermissions
+from weaviate.classes.rbac import Permissions
+
+permissions = [
+    Permissions.cluster(read=True),
+    Permissions.nodes(collection="Test_DevRel", verbosity="verbose", read=True)
+]
+
+admin_client.roles.add_permissions(permissions=permissions, role_name="devrel-admin")
```

Comment:
Does the response provide feedback?

I wonder if we should capture it like:

```python
response = admin_client.roles.add_permissions(permissions=permissions, role_name="devrel-admin")

print(response)
```

Reply:
`add_permissions` doesn't provide feedback no. At least from what I've tested just now